docker/docker-compose.yml

```diff
@@ -20,12 +20,6 @@ services:
     environment:
       NODE_ENV: "${MYAEGEE_ENV}"
       #BUGSNAG_KEY: "${BUGSNAG_KEY_GSUITE_WRAPPER}"
-    labels:
```

Comment:
can you instead only expose /healthcheck endpoint? check out mailer on how it's done here

Reply:
ahhh i forgot, yes! now that i merged the other, definitely